dynamic_instrumentation/src/instrument.rs

```diff
@@ -256,6 +256,16 @@ impl<'tcx> Visitor<'tcx> for InstrumentationAdder<'_, 'tcx> {
                     .debug_mir(location)
                     .add_to(self);
             }
+            Rvalue::Use(Operand::Constant(..))=> {
```

Comment:
```suggestion
            Rvalue::Use(Operand::Constant(..)) => {
```

`cargo fmt --check` is on in CI now.

Reply:
If you merge/rebase to `master`, you can then run `./scripts/run_ci_checks.sh` to check the same things locally.